client/me/purchases/manage-purchase/index.jsx

```diff
@@ -215,7 +215,7 @@ const ManagePurchase = React.createClass( {
 			upgradesActions.addItem( redemptionItem );
 		}
 
-		page( '/checkout/' + this.props.selectedSite.slug );
+		page( '/checkout/' + this.props.selectedSite.domain );
```

Comment:
We should always be using whatever `SitesList` set to `slug` here, which is usually the domain, but switches to a `[site].wordpress.com` if there are conflicts.